src/System.Text.Json/src/System/Text/Json/Node/JsonObject.cs

```diff
@@ -72,7 +70,7 @@ public JsonObject(DuplicatePropertyNameHandling duplicatePropertyNameHandling =
         /// </summary>
         /// <param name="jsonProperty">The property to add.</param>
         /// <exception cref="ArgumentException">
-        ///   Property name to set already exists if handling duplicates is set to <see cref="DuplicatePropertyNameHandling.Error"/>.
+        ///   Property name to set already exists.
```

Comment:
```suggestion
        ///   Property name to add already exists.
```

(I only noticed this because you fixed the next one, but there are others that got missed).